### ISSUE TITLE
File copy error with `include_css` since 4.1.11

### ISSUE THREAD

**Issue by mister-ben** [ankitskvmdam/clean-jsdoc-theme#175]
**Describe the bug**

If `include_css` is used, the file copy fails. This is because `fileCopySync` expects the second arg to be the file path, not destination directory. Reproducible in 4.2.0 and 4.1.11.

https://github.com/ankitskvmdam/clean-jsdoc-theme/blob/c260f86db5720b769ecf00d29cc5c8bacb3a4be3/clean-jsdoc-theme-helper.js#L12

The difference is that in 4.2.0 clean-jsdoc-theme-helper.js imports standard `fs`:

https://github.com/ankitskvmdam/clean-jsdoc-theme/blob/c260f86db5720b769ecf00d29cc5c8bacb3a4be3/clean-jsdoc-theme-helper.js#L2 

Whereas in 4.1.10 a different `fs` is imported:

https://github.com/ankitskvmdam/clean-jsdoc-theme/blob/493e596509ab59147775f49b40be68b27f8d1a6e/publish.js#L4

Updating `copyToOutputFolder` to copy to `path.join(outdir, path.basename(filePathNormalized))` is one option.

**To Reproduce**

- Use `include_css` in `theme_opts` e.g `"include_css": ["./build/docs/styles.css"],`
- Run `jsdoc`

**Expected behavior**

Build completes

**Actual**

```
Error: EISDIR: illegal operation on a directory, copyfile 'build/docs/styles.css' -> 'docs/api'
    at Object.copyFileSync (fs.js:2062:3)
    at copyToOutputFolder (/Users/user/Code/mypkg/node_modules/clean-jsdoc-theme/clean-jsdoc-theme-helper.js:12:8)
    at copyToOutputFolderFromArray (/Users/user/Code/mypkg/node_modules/clean-jsdoc-theme/clean-jsdoc-theme-helper.js:21:13)
    at includeCss (/Users/user/Code/mypkg/node_modules/clean-jsdoc-theme/clean-jsdoc-theme-helper.js:77:21)
```

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser n/a
 - Version Node 14

**Smartphone (please complete the following information):**
 - n/a

**Additional context**

**Comment by ankitskvmdam** [comment]
Thanks for reporting.

**Comment by ankitskvmdam** [comment]
fixed in v4.2.1
